lib/worker/subprocess.js

```diff
@@ -134,23 +143,18 @@ ipc.options.then(async options => {
 		return null;
 	}).filter(provider => provider !== null);
 
-	// Lazily determine support since this prints an experimental warning.
-	let supportsESM = async () => {
-		try {
-			await import('../esm-probe.mjs');
-			supportsESM = async () => true;
-		} catch {
-			supportsESM = async () => false;
-		}
-
-		return supportsESM();
-	};
-
 	let requireFn = require;
+	let isESMSupported;
 	const load = async ref => {
 		for (const extension of extensionsToLoadAsModules) {
 			if (ref.endsWith(`.${extension}`)) {
-				if (await supportsESM()) { // eslint-disable-line no-await-in-loop
+				if (typeof isESMSupported !== 'boolean') {
+					// Lazily determine support since this prints an experimental warning.
+					// eslint-disable-next-line no-await-in-loop
+					isESMSupported = await supportsESM();
+				}
+
+				if (isESMSupported) {
 					return import(pathToFileURL(ref));
```

Comment:
Do we need this `pathToFileURL`?

Reply:
Seems need for Node.js 13 on windows https://github.com/avajs/ava/pull/2404/checks?check_run_id=457268243

Reply:
Yep 😄